src/main/java/hello/Application.java

```diff
@@ -17,6 +17,11 @@ String generateSecretToken() {
 		return Long.toHexString(r.nextLong());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') 'method def' child have incorrect indentation level 16, expected level should be 8. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.37.128.36:9000/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck)